stdeb/command/common.py

```diff
@@ -35,6 +35,7 @@ def initialize_options (self):
         self.with_dh_systemd = False
         self.sign_results = False
         self.ignore_source_changes = False
+        self.compat = 7
```

Comment:
Shouldn't this be a `9` too? Since the default value is now repeated in more than one place please create a global constant for it.

Reply:
> Since the default value is now repeated in more than one place please create a global constant for it.

This comment is still pending. Can you please update the patch to define `DEFAULT_COMPAT = 9` somewhere and use it instead of the hard coded default. And also mention the default value in the help text as well as README. Thanks.

Reply:
@veimox Friendly ping (since I would like to get a new minor release out later this week).